db_stress_tool/db_stress_gflags.cc

```diff
@@ -329,6 +329,9 @@ DEFINE_bool(allow_concurrent_memtable_write, false,
 DEFINE_bool(experimental_allow_mempurge, false,
             "Allow mempurge process to collect memtable garbage bytes.");
 
+DEFINE_string(experimental_mempurge_policy, "kAlternate",
+              "Set garbage collection policy.");
```

Comment:
If you're going to say "garbage collection," please say "memtable garbage collection."